radis/lbl/loader.py

```diff
@@ -1498,6 +1510,52 @@ def fetch_databank(
                     [str(k) for k in self._get_isotope_list(df=df)]
                 )
 
+        elif source == "kurucz":
+            if columns is not None:
+                self.warn("The required columns for Kurucz don't match those of existing moleculear databases, so all columns are being loaded")
+                columns = None
+
+            if memory_mapping_engine == "auto":
+                memory_mapping_engine = "pytables"
+
+            if database != "full":
+                raise ValueError(
+                    f"Got `database={database}`. When fetching Kurucz, only the `database='full'` option is available."
+                )
+
+            # Download, setup local databases, and fetch (use existing if possible)
+
+            if isotope == "all":
+                isotope_list = None
+            else:
+                isotope_list = ",".join([str(k) for k in self._get_isotope_list()])
+            # local_paths, df = fetch_kurucz(
+            #     molecule,
+            #     isotope=isotope_list,
+            #     engine=memory_mapping_engine
+            # )
+            df, local_paths = fetch_kurucz_new(
```

Comment:
should be `fetch_kurucz`, we should not keep old code

Reply:
agreed

Reply:
already fixed: https://github.com/radis/radis/pull/652/commits/addb90c0dc2fe3bdfb915defc6e548802ef0df1f#diff-8694bc131153e428bcabb11b9ff0b5ab0bc832e1b1e5f9431a5c95c936fba48eR21